sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -0,0 +1,460 @@
+-- Query for firefox_ios_derived.feature_usage_metrics_v1
+-- For more information on writing queries see:
+-- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+DECLARE submission_date DATE DEFAULT "2023-12-07";
```

Comment:
```suggestion
```

Reply:
We don't need the DECLARE at all? Across all 4 query files?